Makefile

```diff
@@ -123,6 +124,12 @@ build-local-container: BIN_SUFFIX=-$(GOOS)
 build-local-container: build ## Builds the provisioner container image using locally built binaries
 	$(CONTAINER_RUNTIME) build -f Dockerfile.local -t $(IMAGE) .
 
+kind-load-bundles:
+	$(CONTAINER_RUNTIME) build $(TESTDATA_DIR)/bundles/plain-v0/valid -t testdata/bundles/plain-v0:valid
+	$(CONTAINER_RUNTIME) build $(TESTDATA_DIR)/bundles/plain-v0/invalid-crds-and-crs -t testdata/bundles/plain-v0:invalid-crds-and-crs
+	${KIND} load docker-image testdata/bundles/plain-v0:valid --name $(KIND_CLUSTER_NAME)
```

Comment:
Use $(TESTDATA_DIR) for lines 130 and 131?

Reply:
No need. Those are just "docker host names" and won't need to change if we move the testdata directory.

Reply:
Yikes - I guess I need more coffee in me today. LGTM.